File: program/find-surface-area-of-a-cuboid/FindSurfaceAreaOfACuboid.java
import java.util.Scanner;

public class FindSurfaceAreaOfACuboid {
	
	public static int FindSurfaceAreaOfACuboid(int Height, int Width, int Length) {
		int hw, wl, hl;
		hw = Height*Width;
		wl = Width*Length;
		hl = Height*Length;
		return 2*(hw+wl+hl);
	}

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int height, width, length;
		height = scanner.nextInt();
		width = scanner.nextInt();
		length = scanner.nextInt();
		System.out.println(FindSurfaceAreaOfACuboid(height, width, length));
	}

}

File: program/find-surface-area-of-a-cuboid/FindSurfaceAreaOfACuboid.kt
import java.util.Scanner

fun main() {
    //Input Stream
    val scanner = Scanner(System.`in`)
    print("Enter Length of Cuboid : ")
    val length = scanner.nextInt()
    print("Enter Width of Cuboid : ")
    val width = scanner.nextInt()
    print("Enter Height of Cuboid : ")
    val height = scanner.nextInt()
    //surface Area of Cuboid
    val surfaceAreaofCuboid = 2*(length * width + length * height + width *height)
    //Print Surface Area
    println("Surface Area of Sphere of Cuboid :$surfaceAreaofCuboid")
}
